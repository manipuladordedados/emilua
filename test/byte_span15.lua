local byte_span = require 'byte_span'

local bs = byte_span.new(4, 5)
bs:slice(1, 5)[5] = 33
bs:copy('hello')
print(#bs)
print(bs.capacity)
print(bs)
bs = bs:slice(1, 5)
print(#bs)
print(bs.capacity)
print(bs)
bs:copy('hello')
print(bs)
bs:slice(2):copy(bs)
print(bs)
bs:copy(bs:slice(2))
print(bs)
