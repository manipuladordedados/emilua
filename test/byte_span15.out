4
5
hell
5
5
hell!
hello
hhell
helll
